apps/datahub/src/app/home/news-page/last-created/last-created.component.ts

```diff
@@ -16,11 +16,13 @@ export class LastCreatedComponent implements OnInit {
   ) {}
 
   ngOnInit() {
-    this.searchFacade.setConfigRequestFields({
-      includes: [...ES_SOURCE_BRIEF, 'createDate', 'changeDate'],
-    })
+    this.searchFacade.setConfigRequestFields([
+      ...FIELDS_BRIEF,
```

Comment:
should be pivot format fields

Reply:
This needs a stronger abstraction and will be done later on